examples/benchmark/benchmark.py

```diff
@@ -0,0 +1,113 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import hydra
+import torchdata
+from omegaconf import DictConfig
+from s3torchconnector import S3IterableDataset, S3Reader, S3MapDataset
+from torch.utils.data import DataLoader, default_collate
+from torchdata.datapipes.utils import StreamWrapper
+
+from benchmark_utils import ResourceMonitor
+from models import Entitlement, ViT
+
+
+@hydra.main(version_base=None, config_path="configuration")
+def run_experiment(config: DictConfig):
+    if config.training.model == "entitlement":
+        model = Entitlement()
+    elif config.training.model == "vit":
+        model = ViT(1000, config.checkpoint)
```

Comment:
Why are we hardcoding 1000?

Reply:
Thats number of labels. To be fair, as we are working with synthetic data, any number is fine here. But of-course it would be better to make it parametrised.